src/test/java/Homework21.java

```diff
@@ -0,0 +1,85 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.interactions.Actions;
+import org.openqa.selenium.interactions.SourceType;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+public class Homework21 extends BaseTest{
+    @Test
+    public void renamePlaylist(){
+        String playlistName = "Stay tune" ;
+        String  newName = "Don't feel sorry";
+
+        login("vicplach123@gmail.com", "MEGAdelta06@");
+
+
+        //create playlist
+        createNewPlist(playlistName);
+
+       //check playlist name
+        checkTheNameOfCreatedPlyst(playlistName);
+
+        System.out.println(playlistName);
+
+
+        //rename playlist
+        doubleClickChoosePlaylist();
+
+        enterNewName(newName);
+
+        System.out.println(newName);
+        Assert.assertFalse(playlistName.contains(newName));
+
+
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._